kv/kv.go

```diff
@@ -101,6 +101,10 @@ var ErrNotCommitted = errors.New("this transaction is not committed")
 type Transaction interface {
 	// Get gets the value for key k from KV store.
 	Get(k Key) ([]byte, error)
+	// BatchGet gets a batch of values from KV store.
+	BatchGet(keys []Key) (map[string][]byte, error)
+	// Scan gets values in specific range from KV store.
+	Scan(start, end Key, limit int) (map[string][]byte, error)
```

Comment:
what's the scan range scope? [start, end] or [start, end), you should point it out to let user know.